### PR TITLE
Magicarp now are named after their bolt type

### PR DESCRIPTION
**Pull request by coiax**
> That's a bolt of death magicarp
> 50% magic, 50% carp, 100% horrible.

:cl: coiax
add: Magicarp are now named after the type of magical bolt that they
shoot at people.
/:cl:

Minor buff to Xenobio, as you can now determine a carp's bolt type
without waking it up.

Chaos magicarp are unaffected, and are still called chaos magicarp.

### CODE REVIEW

**Comment by davethwave** [comment]
How would this pr make it oh so much easier to mass produce magicarps? it's not like it increases the chance of it spawning via gold slime or anything.

**Comment by Grazyn** [comment]
It cuts down the time you usually waste on sentience the carp-ask it to shoot something-carp teleports you-go back-kill carp-spawn another-reapeat

**Comment by ExcessiveUseOfCobblestone** [comment]
The real issue is that magicarps exist as a mob drop for gold slimes. This PR just makes it so you can conserve your pinkies and pick your favorite type.

**Comment by davethwave** [comment]
Pick? Its still random and just spawn a monkey or something to test which type it is. This would only shave off a half a minute.

**Comment by ExcessiveUseOfCobblestone** [comment]
Pick as in you don't have to eat a potion to figure out, you get to _pick_ when to use the potion. But yes I guess you could just setup a monkey area.

Again, the root issue is that these things are spawnable by xenobio anyways. That and they are always friendly to humans.

**Comment by kriskog** [comment]
This also means everyone else knows what kind of magicarp you have awakened. I don't know if that is a good thing, as it might cause more random violence against dangerous (but potentially friendly) magicarps.

**Comment by ExcessiveUseOfCobblestone** [comment]
I will accept this PR if and only if chaos magicarp has a chance to become "chaos 2handed zweihander magicarp" and given invulnerability

File: code/modules/events/wizard/magicarp.dm [hunk 1]
```diff
@@ -50,7 +53,9 @@
 	color = "#00FFFF"
 	maxHealth = 75
 	health = 75
+	chaos = TRUE
 
 /mob/living/simple_animal/hostile/carp/ranged/chaos/Shoot()
-	projectiletype = pick(allowed_projectile_types)
+	if(chaos)
+		projectiletype = pick(allowed_projectile_types)
```
Comment (ninjanomnom) [inline code/modules/events/wizard/magicarp.dm]: If you're going to make chaos a var this check needs to be in the type that adds that var

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/events/wizard/magicarp.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/events/wizard/magicarp.dm]: why does the chaos carp need to check for chaos lol

**Comment by Cyberboss** [comment]
Your PR is getting old. Either Travis CI is failing, it is still labelled work in progress (change this by adding `[READY]` to the start of your title), or there are unaddressed change requests. This may be closed soon to free up space on our tracker if it remains idle. In the event that it is, please ask in #coderbus on Rizon for it to be reopened once you have resumed work on it.

**Pull request closed without merging**